src/libs/ReceiptUtils.js

```diff
@@ -34,10 +34,15 @@ function validateReceipt(file) {
  * Grab the appropriate receipt image and thumbnail URIs based on file type
  *
  * @param {String}  path      URI to image, i.e. blob:new.expensify.com/9ef3a018-4067-47c6-b29f-5f1bd35f213d or expensify.com/receipts/w_e616108497ef940b7210ec6beb5a462d01a878f4.jpg
- * @param {String}  filename  of uploaded image or last part of remote URI
+ * @param {String}  [filename]  of uploaded image or last part of remote URI
  * @returns {Object}
  */
-function getThumbnailAndImageURIs(path, filename) {
+function getThumbnailAndImageURIs(path, filename = '') {
+    // For things that don't actually have files we will show the generic receipt icon.
+    if (!filename) {
+        return {thumbnail: null, image: ReceiptGeneric};
+    }
```

Comment:
I'm not really sure about when we use `path` (the case below), but I think this is better placed right after it. So if the function is called with a `path` and no `filename` we still return `return {thumbnail: null, image: path};`